parameter/IntegerParameterType.cpp

```diff
@@ -431,6 +448,16 @@ bool CIntegerParameterType::checkValueAgainstRange(const string &strValue, type
     return true;
 }
 
+// Limit Range accoridng to dynammic
+template <typename type>
+type CIntegerParameterType::LimitValueAgainstRange(type value,
+                                                   type minValue, type maxValue) const
+{
+    if (value > maxValue) return(maxValue);
```

Comment:
As of today .xsd doesn't check the range. So this method allows to have same behavior as when we do not specify Min and Max.
In case of ERROR a tool should be provided in order to check XML mix/max range as I am not sure that the error will be explicit for a full platform XML loading.